src/styles.css

```diff
@@ -56,34 +52,59 @@ body {
 
 #tweet-quote {
   text-decoration: none;
-  color: var(--quoteTextColor);
   font-size: var(--sizeUnit);
+  color: var(--quoteTextColor);
   border-bottom: 
     solid 
-    var(--preHoverColor) 
+    #c9c9c9
     calc(var(--sizeUnit) * 0.1);
 }
 
 i {
-  display: inline-block;
   padding-left: calc(var(--sizeUnit) * 0.1);
 }
 
 #new-quote {
   border: none;
   background-color: var(--quoteTextColor);
-  color: var(--accentColor);
   font-family: "Work Sans";
   font-weight: 600;
   font-size: var(--sizeUnit);
   padding: var(--sizeUnit);
   position: relative;
-  top: -9vw;
+  top: calc(var(--sizeUnit) * -6);
   left: var(--sizeUnit);
   float: right;
 }
 
 #new-quote:hover {
   cursor: pointer;
-  color: #ffffff;
+  color: #ffffff !important; /* since (non-hover) font-color is assigned via JS, need to override it here for the hover state with `!important` */
+}
+
+#legend {
+  display: flex;
+  width: calc(var(--sizeUnit) * 50);
+  gap: var(--sizeUnit);
+}
+
+.legend-label {
+  font-weight: 600;
+  font-size: calc(var(--sizeUnit) * 0.65);
+  line-height: 0.7;
+  flex-grow: 1;
+}
+.legend-label:first-of-type {
+  text-align: right;
+}
+
+#legend-colors {
+  flex-grow: 30;
+  height: calc(var(--sizeUnit) * 0.5);
+  background: linear-gradient(
```

Comment:
Build the legend that maps the range of possible colors to the length of shortest and longest quotes. This uses teh same colorspace (`lab()`) used by colorizer() function to assign each quote a color. It also uses teh same relationship between the `a` and `b` values established in colorizer(): the more saturated `b` is, the less saturated `a` will be, and vice versa, linearly.